http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -100,6 +100,9 @@ protected OidcClientConfiguration internalBuild(final OidcJsonConfiguration oidc
         if (oidcJsonConfiguration.getTokenCookiePath() != null) {
             oidcClientConfiguration.setOidcStateCookiePath(oidcJsonConfiguration.getTokenCookiePath());
         }
+        if(oidcJsonConfiguration.getScope() != null) {
```

Comment:
Just a minor style thing to be consistent with the rest of the code, we normally include a space in between `if` and the opening bracket, i.e.,

`if (`